### ISSUE TITLE
Pendulum calibration failed! Reason: hit the right switch while attempting to reach the left one

### ISSUE THREAD

**Issue by ss555** [francofusco/pendule_pi#24]
when attempting to run
`sudo ./low_level_interface ../pendule_pi_config.yaml` from the build
following is shown on the console :
```
Calibrating pendulum
terminate called after throwing an instance of 'pendule_pi::Pendule::CalibrationFailed'
  what():  Pendulum calibration failed! Reason: hit the right switch while attempting to reach the left one
```

Cause: motor direction inversed or switch pins.
observation: when we test the motor with +pwm, the positional encoder increments +

workaround(tested): inverse pins for switches
Add in doc: propose the solution of yaml file

P.S additional feature to be added that in the case if the cart hits the switch, it should change the dir for some moment, so that we shouldn't move the cart by hand to deactivate the switch.

**Comment by francofusco** [comment]
Yeah, the solution is to use the yaml file to properly configure the switches 😊 since I introduced the new config mechanism in the last commits, I have yet to document it properly, my bad as usual... 😥

**Comment by francofusco** [comment]
> P.S additional feature to be added that in the case if the cart hits the switch, it should change the dir for some moment, so that we shouldn't move the cart by hand to deactivate the switch.

I would rather not do that, just because if something fails, I just want all components to be shut down. You can still move the base "manually" with the assistant, so I don't think it's a big deal :wink:
